.github/ISSUE_TEMPLATE/content-template.yaml

```diff
@@ -1,7 +1,6 @@
-name: Request new or edited content
+name: Request New or Edited Content
 description: Submit your requests and suggestions to help us improve!
-title: '[Content] '
```

Comment:
Was taking this line out intentional?

Reply:
yes

Reply:
Doesn't the title pre-populate the name of the issue? I don't think "titles" and "labels" are mutually exclusive but if you prefer not pre-formatting titles at all for folks posting issues, I guess it works the same. In my experience, people have a tendency to use non-descriptive titles that don't help explain what the issue is about which is the only reason I'd push for it.

Reply:
Yes: no prefill now.
The labels I think will cover broad categories that users are going for, it will be up to us to edit the title as we see fit to help us